dev/com.ibm.ws.security.wim.core_fat/publish/servers/com.ibm.ws.security.wim.core.fat.mod/.gitignore

```diff
@@ -0,0 +1 @@
+/dropins
```

Comment:
We should be building in the autofvt build directory, so not sure we should need this.

Reply:
Removed